wiki/osu!supporter/en.md

```diff
@@ -57,6 +57,15 @@ Supporters can further customise their profile by setting a cover image:
 
 Once osu!supporter expires, the cover will remain, but cannot be changed.
 
+### Profile colour
+
+Akin to profile covers, supporters can alter the colour of their profile:
+
+- Press the pencil icon in the lower right corner of the profile cover.
+- Adjusting the profile colour alters the colour of almost everything, including the banner, buttons, links, and text.
+
+Once osu!supporter expires, the colour customizations will remain, but cannot be changed.
```

Comment:
I assume this is true but can someone else confirm just to be sure

Reply:
it can be changed back to default but not to any other custom color

Reply:
> it can be changed back to default but not to any other custom color

thanks for the confirmation